k8s/externalsnapshotter/volumesnapshotcontents.go

```diff
@@ -0,0 +1,60 @@
+package externalsnapshotter
+
+import (
+	"github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SnapshotContentOps is an interface to perform k8s VolumeSnapshotContent operations
+type SnapshotContentOps interface {
+	// CreateSnapshotContent creates the given snapshot
```

Comment:
nit: .. given snapshot content
(same for all comments below)

Reply:
Updated.